regression/goto-analyzer/pruning-lt-nondet/test-constants.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--variable-sensitivity --vsd-values constants --verify
+^EXIT=0$
+^SIGNAL=0$
+\[main\.assertion\.1\] line .* assertion x < 20: SUCCESS
+
```

Comment:
Hello @jezhiggins, bot this and `pruning-lt-nondet-2/test-constants.desc` contain an empty line.

This looks like at least in the windows system, it fails to much as a regex. Can you delete these empty lines and see if it behaves?

I'm surprised how this is not a problem on other platforms though.

Reply:
You're a superstar! That's got it sorted, thank you.